package.json

```diff
@@ -12,10 +12,10 @@
     "stickyfilljs": "^2.1.0",
     "toaster": "^0.1.2",
     "turbolinks": "^5.2.0",
-    "validate.js": "^0.13.1"
+    "validate.js": "^0.13.1",
+    "babel-plugin-prismjs": "^2.0.1"
```

Comment:
@TheOdinProject/odinmods  This is ugly 🙈 but not sure why the Heroku ruby buildpack does not install both `dependencies` and `devDependencies` as [Heroku Node.js buildpack](https://devcenter.heroku.com/articles/nodejs-support#package-installation) does.

If anyone knows a better solution, do let me know!

Reply:
Can you put this fix in a separate PR please Arun? We can't deploy without it.

Reply:
Sure.

Reply:
Just opened https://github.com/TheOdinProject/theodinproject/pull/1123 addressing this.

Reply:
Thanks @arku 💪